lib/api/statistics.js

```diff
@@ -3,6 +3,10 @@
  * @fileOverview Module for interacting with the statistcs.json endpoint.
  *
  */
+
+// Local
+const Promise = require('../polyfills/promise');
```

Comment:
I'd like to actually ponyfill fetch within this repo also, and also in a non-polluting way, and then import that ponyfill wherever we're using fetch internally here... That would just be done on an upcoming ticket.